deploy/kubernetes/helm/templates/NOTES.txt

```diff
@@ -0,0 +1,26 @@
+1. You can check the status of HDFS by running this command:
```

Comment:
Do we need this file?

Reply:
Yes indeed - it gives some overview of what has been installed.